content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
+* Before you create a webhook, [create any secrets it needs]({{< relref "#add-a-secret" >}}).
+{{% /alert %}}
+
+This section shows how to configure a webhook's URL, as well as any access tokens and secrets it requires.
+
+1. If the webhook requires any access tokens or other sensitive strings, create a secret in W&B for each sensitive string, and make a note of the secret's name.
+1. Log in to W&B and go to the **Settings** page.
+1. In the **Webhooks** section, click **New webhook**.
+1. Provide a name for the webhook. 
+1. Provide the endpoint URL for the webhook.
+1. If the webhook authenticates using an access token, set **Access token** to the name of the secret that contains the access token. When you configure an automation that uses this webhook, the access token will be available in the `$ACCESS_TOKEN` environment variable, and the HTTP header will have `Authorization: Bearer $ACCESS_TOKEN`.
+1. If the webhook validates its payload using a sensitive string, set **Secret** to the name of the secret that contains the sensitive string.
+1. Click **Test** to test the webhook. Optionally, provide a payload to test. Any payload you specify in this step does not persist, and will need to be specified when you [create the webhook automation]({{< relref "#create-webhook-automation" >}}).
+
+{{% alert %}}
+See the [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}) section to view where the secret and access token are specified in
+the POST request.
+{{% /alert %}}
+
+Now you can [create an automation that uses the webhook]({{< relref "#create-webhook-automation" >}}).
+
+#### Create the automation {#create-webhook-automation}
+1. Log in to W&B and go to the project page.
+1. In the sidebar, click **Automations**.
+1. Click **Create automation**.
+1. Choose the **Event** which triggers the automation. If applicable, provide options that are specific to the event type. If your project has no registries, registry events will not be available. Click **Next step**.
+1. Select the team where you added the webhook.
+1. Set **Action type** to **Webhook**, then select the webhook.
+1. Provide the payload for the webhook in **Payload**. Refer to the reference for varialbles that you can use. For details, refer to the [Example webhook payloads]({{< relref "#example-webhook-payloads" >}}) section.
+1. Click **Next step**.
+1. Provide a name for the automation. Optionally, provide a description.
+1. Click **Create automation**.
+
+### Example webhook payloads
+
+The following tabs demonstrate example payloads based on common use cases. Within the examples they reference the following keys to refer to condition objects in the payload parameters:
+* `${event_type}` Refers to the type of event that triggered the action.
+* `${event_author}` Refers to the user that triggered the action.
+* `${artifact_version}` Refers to the specific artifact version that triggered the action. Passed as an artifact instance.
```

Comment:
We need to answer the question: what does it mean that something is 'passed as an artifact instance'?

Reply:
Good idea. This content is copied from where it lives in the current doc, not net new in this PR.

Reply:
https://docs.wandb.ai/ref/python/artifact/